exporters/prometheus/confg_test.go

```diff
@@ -58,3 +80,33 @@ func TestNewConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestConfigManualReaderOptions(t *testing.T) {
+	aggregationSelector := func(view.InstrumentKind) aggregation.Aggregation { return nil }
+
+	testCases := []struct {
+		name       string
+		config     config
+		wantOption []metric.ManualReaderOption
```

Comment:
This does not test the value of these options. Is there any reason this doesn't instead accept just an expected option length here?

Reply:
I did this to try and compare equality, but I haven't been able to figure out how.  I think for now I will move it to count and roll it into the test above.